src/Build/Microsoft.Build.csproj

```diff
@@ -42,6 +39,11 @@
     <PackageReference Include="Microsoft.BuildXL.Processes" Condition="'$(FeatureReportFileAccesses)' == 'true'" PrivateAssets="all" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'">
+    <PackageReference Include="System.Collections.Immutable" />
+    <PackageReference Include="System.Threading.Tasks.Dataflow" />
```

Comment:
Nit: can you combine these into the existing group on line 48 (in the "before" file)?

Reply:
good spot, i missed that in my rush to get it done.